app/src/main/java/com/codefororlando/petadoption/about/AboutActivity.kt

```diff
@@ -22,55 +24,55 @@ class AboutActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_about)
-        val toolbar = findViewById(R.id.toolbar) as Toolbar
+        val toolbar = findViewById<Toolbar>(R.id.toolbar)
```

Comment:
It's generally a lot cleaner to use generic inference rather than explicitly defining the generic and relying on type inference such as

```kotlin
val toolbar: Toolbar = findViewById(R.id.toolbar)
```

In at least one use case you ended up specifying it both ways.